app/permissions/decidim/decidim_awesome/admin/permissions.rb

```diff
@@ -13,8 +13,19 @@ def permissions
 
           toggle_allow(config_enabled?(permission_action.subject)) if permission_action.action == :edit_config
 
+          case permission_action.action
+          when :index
+            admin_accountability_enabled?
+          end
+
```

Comment:
```suggestion
          case permission_action.subject
          when :admin_accountability
            admin_accountability_enabled?
          end
```

I think like this makes more sense